### PR TITLE
Feature cha 7055 swagger

### PR DESCRIPTION
**Pull request by Sacrelicious**

### CODE REVIEW

File: Raven/Controllers/AdvantageController.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        public async Task<IActionResult> GetEmployers()
+        {
+            var employers = await _advantageService.GetAllAdvantageEmployers();
+
+            return Ok(employers);
+        }
+
+        [HttpGet]
+        [ProducesResponseType(typeof(ICollection<Employer>), 200)]
+        [Route("productsForEmployer/{employerIdentifier}")]
```
Comment (jturinetti) [inline Raven/Controllers/AdvantageController.cs]: I prefer `api/advantage/employers/{employerGuid}/plans`

File: Raven/Controllers/AdvantageController.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    {
+        private readonly IAdvantageService _advantageService;
+
+        public AdvantageController(IAdvantageService advantageService)
+        {
+            _advantageService = advantageService;
+        }
+
+        [HttpGet]
+        [ProducesResponseType(typeof(ICollection<Employer>), 200)]
```
Comment (jturinetti) [inline Raven/Controllers/AdvantageController.cs]: `api/advantage/employers`

File: Raven/Models/Employer.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Raven.Models
+{
+    public class Employer
+    {
+        public Guid EmployerIdentifier { get; set; }
+        public string Name { get; set; }
+        public string Id { get; set; }
```
Comment (jturinetti) [inline Raven/Models/Employer.cs]: Is `Id` descriptive enough?

File: Raven/Models/Product.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Raven.Models
+{
+    public class Product
+    {
+        public string Name { get; set; }
+        public string Carrier { get; set; }
+        public string PlanId { get; set; }
+        public string NetworkId { get; set; }
```
Comment (jturinetti) [inline Raven/Models/Product.cs]: `NetworkCode`

File: Raven/Models/Employer.cs (hunk 3 above)
Comment (Sacrelicious) [reply Raven/Models/Employer.cs]: (shrug) it seems weird to say "EmployerId" when it's on the employer object?

**Pull request merged**